lib/primer/open_project/forms/dsl/input_methods.rb

```diff
@@ -37,6 +37,10 @@ def html_content(**, &)
             add_input HtmlContentInput.new(builder:, form:, **, &)
           end
 
+          def pattern_input(**, &)
+            add_input PatternInput.new(builder:, form:, **decorate_options(**), &)
```

Comment:
Could you add an example to `spec/lib/primer/open_project/forms/dsl/input_methods_spec.rb` to cover this? - it's kind of a stupid spec, but for the moment, helps us ensure that we're decorating options consistently.